app/views/layouts/submissions/editor/_problem_form.html.erb

```diff
@@ -22,14 +22,17 @@
       <input type="file" id="upload" class="upload submission-control"/>
     <% end %>
 
-    <%= form_for([@exercise, @solution], remote: true, html: {role: 'form'}) do |f| %>
+    <%= form_for([@exercise, @solution], remote: true, html: {role: 'form', class: 'editor-overlap'}) do |f| %>
       <% if editor_mode == :visible %>
         <div class="field form-group">
           <%= f.editor :content, 'dynamic',
                        class: 'form-control editor',
-                       value: @default_content,
+                       value: @current_content,
                        style: 'display: none' %>
         </div>
+        <div class="editor-overlapped">
+          <a class="editor-trash"><%= fa_icon('trash-o', text: t(:reset))%></a>
```

Comment:
perhaps we could rename this style to `editor-reset` to be consistent with names in model.